src/rust/ensogl/lib/components/src/list_view/entry/list.rs

```diff
@@ -0,0 +1,211 @@
+//! A module defining entry [`List`] structure: a view of ListView entries arranged in column.
+
+use crate::prelude::*;
+
+use crate::list_view::entry;
+use crate::list_view::entry::Entry;
+
+use ensogl_core::application::Application;
+use ensogl_core::display;
+use ensogl_core::display::scene::layer::LayerId;
+
+
+
+// ======================
+// === DisplayedEntry ===
+// ======================
+
+/// A displayed entry in select component.
+///
+/// The Display Object position of this component is docked to the middle of left entry's boundary.
+/// It differs from usual behaviour of EnsoGl components, but makes the entries alignment much
```

Comment:
Its coold ocs here, good that you describe the difference. Would you also describe shortly why it makes it easier?